slackbot-guysgals-reasoning.md

```diff
@@ -0,0 +1,34 @@
+# Why the Auto Response?
+
+The ZATech community has some people who feel excluded by certain words. In particular there was a point raised in #diversity about the word `guys`.
+
+That's not the MAIN reason why the slackbot auto response exists though. There's a bigger purpose. The purpose is to try and get people to be more aware about the language they use and to try and be more inclusive towards other humans. Regardless of Class, Race, Gender, Sex or other divides that humans love putting up.
+
+We want to be a loving and inclusive community, and we know that the community is made up of the individuals. Thus if the individuals are made more aware of how better to be inclusive to the group, the community as a whole becomes more inclusive.
+
+
+# What we're not trying to do
+
+This is NOT about policing language or stopping freedom of speech.
+It's about getting people to be a little bit more aware of the language they use and the implications thereof.
```

Comment:
"It's about getting people to be a little bit more aware of the language they use and their implications." doesn't sound right. I think I'm misunderstanding what you're trying to say :)